modules/op-resolver-community-tasks-ref.adoc

```diff
@@ -0,0 +1,411 @@
+        kind: Task
+        name: kubeconfig-creator
+      workspaces:
+        - name: output
+          workspace: shared-workspace
+  #...
+----
+
+.Supported parameters for the `kubeconfig-creator` community task
+[cols="1,2,1",options="header"]
+|===
+| Parameter | Description | Default value
+|`name` |The name of the cluster to access. |
+|`url` |The address of the cluster to access. |
+|`username` |The username for basic authentication to the cluster. |
+|`password` |The password for basic authentication to the cluster. |`""`
+|`cadata` |Contains PEM-encoded certificate authority (CA) certificates. |`""`
+|`clientKeyData` |Contains PEM-encoded data from a client key file for TLS. |`""`
+|`clientCertificateData` |Contains PEM-encoded data from a client certification file for TLS. |`""`
+|`namespace` |The default namespace to use on unspecified requests. |`""`
+|`token` |The bearer token for authentication to the cluster. |`""`
+|`insecure` |To indicate whether a server should be accessed without verifying the TLS certificate. |`false`
+|===
+
+.Supported workspaces for the `kubeconfig-creator` community task
+[cols="1,3",options="header"]
+|===
+| Workspace | Description
+| `output` | The workspace where the `kubeconfig` file is stored by the `kubeconfig-creator` task.
+|===
+
+[discrete]
+[id="op-ctaskref-pull-request_{context}"]
+== pull-request
+
+You can use the `pull-request` community task to interact with a source control management (SCM) system through an abstracted interface.
+
+This community task works with both public SCM instances and self-hosted, enterprise GitHub, or GitLab instances. 
+
+In download mode, this task populates the `pr` workspace with the state of the existing pull request, including the `.MANIFEST` file. 
+In upload mode, this task compares the contents of the `pr` workspace with the `.MANIFEST` file, if it exists.
+
+.Example usage of the `pull-request` community task
+[source,yaml,subs="attributes+"]
+----
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: pull-request
+spec:
+spec:
+  tasks:
+    - name: pull-request
+      params:
+        - name: mode
+          value: upload
+        - name: url
+          value: example_pull_request_URL
+        - name: provider
+          value: github
+        - name: secret-key-ref
+          value: example_secret
+        - name: insecure-skip-tls-verify
+          value: 'false'
+      taskRef:
+        kind: Task
+        name: pull-request
+      workspaces:
+        - name: pr
+          workspace: shared-workspace
+  #...
+----
+
+.Supported parameters for the `pull-request` community task
+[cols="1,2,1",options="header"]
+|===
+| Parameter | Description | Default value
+|`mode` |If set to `download`, the state of the pull request at `url` is fetched. If set to `upload` then the pull request at `url` is updated. |
+|`url` |The URL of the pull request. |
+|`provider` |The type of the SCM system. The supported values are `github` or `gitlab`. |
```

Comment:
As before, I suggest removing "The" from the start of the descriptions of all parameters.